src/test/java/apicurito/tests/utils/slenide/VersionUtils.java

```diff
@@ -0,0 +1,31 @@
+package apicurito.tests.utils.slenide;
```

Comment:
After creating new methods for creating and checking media types this class is no longer needed? Media types exist only in OpenAPI v3.

Reply:
I'm not sure about that, the media types are only difference I noticed, but that does not mean there aren't any more differences.